crud.py

```diff
@@ -0,0 +1,86 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(db: Session):
+    return db.query(models.Author).all()

```

Comment:
You forgot to add pagination per requirements for authors and books. Read the checklist ;)

Reply:
Add pagination